spec/license_spec.rb

```diff
@@ -1,35 +1,32 @@
 require 'spec_helper'
 
-# List of licenses that need not be OSI approved
-LICENSE_WHITELIST = %w[
-  unlicense
-  cc0-1.0
-  wtfpl
-  bsd-3-clause-clear
```

Comment:
This is also handled explicitly below. Not sure why the whitelist is needed at all.